ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -64,6 +64,11 @@ const formSchema = z.object({
       skipExcluded: z.boolean(),
       enabledPackageManagers: z.array(z.string()),
     }),
+    advisor: z.object({
```

Comment:
Commit message:

> The default advisors are OSV and VulnerableCode, 

That's not strictly correct; ORT core itself has no default advisors. It's just the Jenkins example pipeline that configures these by default. And this makes sense, but we should change the word to:

"Enable the advisor job with OSV and VulnerableCode advisors by default to get some meaningful vulnerability results."

Or so.